README.md

```
@@ -169,7 +192,7 @@ services:
 - [ ] Display event time in notifications
 - [ ] Online/offline time count
 - [ ] Active bot mode
-     - [ ] Request current state of services
+     - [x] Request current state of services
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 5

(MD007, ul-indent)